ong/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+
+# Create your models here.
+class Ongs(models.Model):
+    name= models.CharField(max_length= 100, null= True, unique= True)
```

Comment:
Since we are using `uuid` to define our primary key, i believe we have to manually imply that here in the model as well.

Something like `ong_uuid = models.UUIDField(primary_key=True, default=uuid4, editable=False)`

Reply:
Done